src/Symfony/Component/Scheduler/Export/ExporterInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Export;
+
+use Symfony\Component\Scheduler\Task\TaskInterface;
+use Symfony\Component\Scheduler\Task\TaskListInterface;
+
+/**
+ * @author Guillaume Loulier <contact@guillaumeloulier.fr>
+ */
+interface ExporterInterface
+{
+    public function exportSingle(TaskInterface $task, string $filePath, string $format): void;
```

Comment:
i  would do the opposite, export() => one and exportMultiple|Many() => multiple|many

Reply:
Renamed